src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/api/controller/probation/ProbationAPIController.kt

```diff
@@ -45,16 +53,24 @@ class ProbationAPIController(
     @PathVariable(name = "defendantId") defendantId: String,
     @RequestBody probationCase: ProbationCase,
   ) {
-    probationCase.createPerson()
+    val defendant: PersonEntity = retrieveDefendant(defendantId)
+    val offender: PersonEntity = probationCase.createPersonWithoutLinkingAndClustering()
```

Comment:
can't this just be allowed to join whatever cluster it wants?
99/100 it will already exist anyway I think